test/e2e/gvrs.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2021 Banzai Cloud.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import "k8s.io/apimachinery/pkg/runtime/schema"
+
+var serviceGVR = schema.GroupVersionResource{
```

Comment:
Aren't these defined somewhere already maybe?

Reply:
I couldn't find any such definitions, but there are duplicated GVRs in istio-operator, so I'm gonna clean this up now